request.go

```diff
@@ -307,15 +305,20 @@ func (ctx *Context) UserValue(name string) any {
 	return ctx.context.UserValue(name)
 }
 
-// Returns Paginator with page size from query parameters
+// Sets default size of the page if not specified in the http request.
+func (ctx *Context) SetPagingDefaultPageSize(pageSize int) {
```

Comment:
This method should not be added to context as it sets global default not related to context